exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,8 @@
-const name = " Daniel  ";
+const myName = "Daniel";
+let greetings = "My name is";
+let nameLength = myName.length;
+let message = `${greetings} ${myName} & my name is ${nameLength} characters long`;
```

Comment:
That a tidy code. I just have one suggestion ,in this case if greeting is a fix string which will not be change, it can be like this : `$My name is ${myName} & my name is ${nameLength} characters long` insead of ${greeting} as a variable